owncloudApp/src/main/java/com/owncloud/android/presentation/thumbnails/ThumbnailsRequester.kt

```diff
@@ -0,0 +1,117 @@
+package com.owncloud.android.presentation.thumbnails
```

Comment:
Let's add the copyright header to this class and, in this case, Abel's name since this was his creation 😁